app/utils/versionChecker.js

```diff
@@ -1,3 +1,5 @@
+const fetch = require('node-fetch');
```

Comment:
was there some issue with fetch, that you added is as a dependency?

Reply:
Yes, there was an issue. It was not able to find fetch. So, i added it.

Reply:
Yes there was an issue. It was not able find fetch. So i added it.